rebar.config

```diff
@@ -66,14 +66,17 @@
                    {ref, "0a82f0f"}}},
 
         {aebytecode, {git, "https://github.com/aeternity/aebytecode.git",
-                      {ref, "2a9035d"}}},
+                      {ref, "7c6a80f"}}},
```

Comment:
Isn't it going to change anyway after merge?

Reply:
Yes.